routing/routing.xml

```diff
@@ -23,29 +23,336 @@
+        
+        A value of "-1" is given if the obstacle can not be passed;
+        A value of "0" is given if the obstacle can be passed without problem.
+        A penalty in seconds is given if the obstacle slows you down. (E.g. a kerb might slow dow 
+        
+        This value is only used by the routing engine
+        
+
+        E.g. a kerb is a showstopper for a wheelchair
+        <routingProfile name="wheelchair">
+            <point attribute="obstacle>
+                <select value="-1" t="barrier" v="kerb" />
+            </point>
+        </routingProfile>    
+        
+        
+        E.g. a kerb causes a slight delay for cyclists
+        <routingProfile name="bicycle">
+            <point attribute="obstacle>
+                <select value="5" t="barrier" v="kerb" />
+            </point>
+        </routingProfile>  
+        
+
+    ## 6) Obstacle_time (specified on points, penalty time in seconds)
+
+        Gives a slowdown time for an obstacle in seconds.
+        In contract to `obstacle`, this value is only used to calculate an ETA and show it on the UI once the route is calculated.
+        In other words, the values here do not influence routing.
+        
+        
+
+    ## 7) Penalty_transition (specified on ways, penalty time in seconds)
+
+        When driving a car, changing from a motorway to a trunk slows you down: you should already slow down 500 meters before the trunk.
+        This has a penalty which is taken into account by (only) the navigation algorithm.
+        This is done by taking the difference between two penalties.
+        The formula used is `maximum(0, penalty_new_way - penalty_current_way)`.
+        
+        E.g. 
+        <way attribute="penalty_transition">
+            <select value="0" t="highway" v="motorway"/>
+            <select value="5" t="highway" v="motorway_link"/>
+            <select value="20" t="highway" v="tertiary"/>
+        </way>
+        
+        
+        This means that, going from a `motorway_link` to a `tertiary`, the penalty given will be
+        `maximum(0, 20 - 5)` or `15 seconds`.
+        
+        On the other hand, going from motorway to motorway incurs no penalty, neither does going from a motorway_link to a motorway.
+
+    ## 8) obstacle_srtm_alt_speed (specified on ways, in seconds of penalty for each height meter)
+    
+	Specifies a penalty for steep ways. Optional. Note that the value of a key can be used as value. For more info about this, see the section about parameters.
+
+	E.g.:
+
+
+  		<gt value1="-60" value2=":incline">
+  			<select value="35"/>
+  		</gt>
+  		<gt value1="-17" value2=":incline">
+  			<select value="10"/>
+  		</gt>
+  		<gt value1="-3" value2=":incline">
+  			<select value="1"/>
+  		</gt>
+  		<gt value1="3" value2=":incline">
+  			<select value="0"/>
+  		</gt>
+
+  		<gt value1="17" value2=":incline">
+  			<select value="1"/>
+  		</gt>
+  		<gt value1="13" value2=":incline">
```

Comment:
It does not make much sense to have 13 after 17, because 17 will always match before 13, so that 13 is not reachable.

Reply:
Good catch, fixed